scripts/just.config.ts

```diff
@@ -22,7 +24,7 @@ export const build = () => {
   const baseEsbuildOptions: EsbuildBuildOptions = {
     entryPoints: glob.sync(["src/**/*.{ts,tsx}", "!src/**/__tests__/**"]),
     outdir: "lib",
-    target: "es6",
+    target: "es2018",
```

Comment:
nitpick: shouldn't we produce change files all packages?

Reply:
I couldn't find a quick way to bump all packages with beachball. Do you know one?

Reply:
you can try `yarn beachball change --all`

Reply:
Great, thanks! I'll do it in the follow up.